src/main/java/Cleese/Task.java

```diff
@@ -0,0 +1,32 @@
+package Cleese;
+
+public class Task {
```

Comment:
Since this class is never used as an object itself, can consider making this an abstract class instead and have the other classes inherit from it. In that case, the ```toStorageString()``` method can be an abstract method and there will be no need for a ```"placeholder"``` in it.